### DISCUSSION TITLE
Add additional icons for Microsoft Products

### DISCUSSION THREAD

**Discussion by AnthonyPhan** [microsoft/fluentui#25038] [Ideas]
Hi All,

It would be great if the FluientUi team could add some additional icons to represent branding for some of Microsoft products including:

- Microsoft PowerBi
- Microsoft Power Automate
- SharePoint

Many of these icons are already available here:
https://developer.microsoft.com/en-us/fluentui#/styles/web/icons

**Comment by Remi749** [comment]
This is much needed!

**Comment by mbice** [comment]
We would love to see this added to V8 React icons, too. Particular the Viva product suite of icons (e.g. VivaEngage). Thanks!
